ember-bootstrap/addon/utils/embroider-config.d.ts

```diff
@@ -0,0 +1,4 @@
+export default interface EmbroiderConfig {
```

Comment:
I'm not sure if `addon/utils/embroider-config.d.ts` is the right location for this interface. Sharing best practices is highly appreciated. :smile_cat: